app/server/appsmith-server/src/main/java/com/appsmith/server/controllers/ce/WorkspaceControllerCE.java

```diff
@@ -82,4 +82,12 @@ public Mono<ResponseDTO<Workspace>> deleteLogo(@PathVariable String workspaceId)
         return service.deleteLogo(workspaceId)
                 .map(workspace -> new ResponseDTO<>(HttpStatus.OK.value(), workspace, null));
     }
+
+    @JsonView(Views.Public.class)
+    @GetMapping("/home")
+    public Mono<ResponseDTO<List<Workspace>>> getUserWorkspacesByRecentlyUsedOrder() {
```

Comment:
The naming convention is tightly linked to current UX for recently used. This can change anytime. Please rename to possibly display the use case (aka workspacesForHome)
```suggestion
    public Mono<ResponseDTO<List<Workspace>>> getUserWorkspacesByRecentlyUsedOrder() {
```

Reply:
Aligned for controller, do we want to update the service level methods as well?

Reply:
Controller is good enough for now.